concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1485,7 +1485,7 @@ Hale-1973-1798-1483	Role_and_Social_Structure-137	marry	v	11b2.17	203	MARRY
 Hale-1973-1798-1484	Role_and_Social_Structure-138	give in marriage	v	11b2.18		
 Hale-1973-1798-1485	Role_and_Social_Structure-139	settle (marriage)	v	11b2.19		
 Hale-1973-1798-1486	Role_and_Social_Structure-140	divorce	v	11b2.20	3672	DIVORCE (SEPARATE)
-Hale-1973-1798-1487	Role_and_Social_Structure-141	desert	v	11b2.21		
+Hale-1973-1798-1487	Role_and_Social_Structure-141	desert	v	11b2.21	3687	DESERT
```

Comment:
From the context I guess "desert" here means "to [withdraw](https://www.merriam-webster.com/dictionary/withdraw) from or leave usually without intent to return", whereas `3687 DESERT` is a noun: https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L3673